x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel_wrapper.tsx

```diff
@@ -199,10 +237,30 @@ export function WorkspacePanelWrapper({
         className={classNames('lnsWorkspacePanelWrapper stretch-for-sharing', {
           'lnsWorkspacePanelWrapper--fullscreen': isFullscreen,
         })}
+        css={{ height: '100%' }}
         color="transparent"
       >
-        <WorkspaceTitle />
-        {children}
+        <EuiFlexGroup
+          gutterSize="none"
+          alignItems="center"
+          justifyContent="center"
+          direction="column"
+          css={{ height: '100%' }}
+        >
+          <EuiFlexItem
+            data-test-subj="lnsWorkspacePanelWrapper__innerContent"
+            grow={false}
+            css={{
+              flexGrow: 0,
+              height: '100%',
+              width: '100%',
+              ...visDimensionsCSS,
+            }}
```

Comment:
Same question here regarding use of Emotion.

Reply:
the `css` prop is provided by Emotion, so we're already using it